rasa/nlu/test.py

```diff
@@ -898,6 +912,79 @@ def compute_metrics(
     return (intent_metrics, entity_metrics, intent_results, entity_results)
 
 
+def compare_nlu(
+    configs: List[Text],
+    data: TrainingData,
+    exclusion_percentages: List[int],
+    micros: Dict[Text, Any],
```

Comment:
what is `micros`?

Reply:
It is a list of lists containing the f1 scores for every run and model. You will have number of f1 scores per run equal to the number of percentages you define.
Will rename it to `fscore_results`.